tools/vpc-flowlogs-analysis/variables.tf

```
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2020 Google LLC
```

double check date in all license headers.
```suggestion
 * Copyright 2021 Google LLC
```